master/getting-started/rkt/installation/manual.md

```diff
@@ -111,13 +108,13 @@ chmod +x /opt/bin/calicoctl
 See the [`calicoctl` documentation]({{site.baseurl}}/{{page.version}}/reference/calicoctl/)
 for more information on this CLI tool.
 
-## Installing the Calico CNI plugins
- 
-You can configure multiple networks when using rkt. Each network is represented by a configuration file in
-`/etc/rkt/net.d/`.   To use Calico networking and policy, create a network that uses the Calico
-CNI binaries for handling networking, policy and IP address management.
+## Installing Calico as a CNI plugin
 
-### Install the Calico plugins
+To install Calico as a CNI plugin used by rkt, we need to first install the
+actual plugin binaries, and then once installed create any CNI networks that you
+require with the appropriate Calico CNI plugin references.
+
+### Install the Calico plugin binaries
```

Comment:
Can you add an extra # to make it 4? Renders better on the site if you skip from 2 to 4.